### DISCUSSION TITLE
Download offline maps (smartphone/tablet)

### DISCUSSION THREAD

**Discussion by Maverikk65** [devemux86/cruiser#677] [Q&A] [answered]
Hello, I am a newbie... Well, I used Kurviger already and Copilot (from Trimble, which is totally broken now...). So I have to search for a new navigation, I liked the Copilot a lot... 

However, I had serious issues with download of offline maps. On my smartphone I was offered small tiles for Germany (I needed 3 of them) and they were okay. But on the tablet (with which I drive mostly) I was only offered the Germany as one piece of 1.8GB and it did not download. I was in a hotel but the wifi was not too bad... However I had to try it 5-6 times before I (hopefully finally) succeeded with the download. 

Is it a bug or a feature that I cannot download tiles on the tablet? 

I did a test drive yesterday and was not fully satisfied at once --- so beware for some questions or feature requests ;-) 
VG, NG.

**Comment by devemux86** [comment, marked as answer]
Hi @Maverikk65,

To download offline maps you can use the menu "Map | Offline maps" and in the first tab:

- Germany from Mapsforge is 2.8 GB (image 1).
You need fast internet and keep the app and screen open to complete the download and install the map.

- Or you can download the smaller maps of Germany folder faster (image 2).

<img src="https://github.com/user-attachments/assets/2771f84c-1b63-4234-a2c9-d9de1439dee9" width="50%"/><img src="https://github.com/user-attachments/assets/fb1f1cda-788f-4008-ae73-fd6f401ce45b" width="50%"/>

**Reply by Maverikk65** [reply]
OMG --- I see my error. Many tnx for pointing me there... On the smartphone I only saw the Germany folder and downloaded the regions. On the tablet, with the much larger screen, I only saw the map (and overlooked the folder) and thought I have to download the 1.8 GB map as a whole...
Tnx! Danke, VG, NG.

**Comment by MotoUKRider** [comment]
Hello @Maverikk65 , former CoPilot user here - You certainly won't miss having to convert routes and copying files into strange folders if you want to import routes or favourites, Cruiser does it much easily.  I also liked CoPilot because of its ability to handle stop/start routes with a photo/rest stop on the countdown, and it was *reasonable* at making a route on the fly, but again - Cruiser does it much more elegantly.

There are some things you might miss, but Cruiser isn't a *"Get me there as quickly as possible by telling me about traffic"* kinda App, there's plenty like that - But it does handle "fun" routes very well, and complex multi-stop tours, different routing profiles (ie: fast or curvy) *within the same* route are just a few features.

I use Garmin BaseCamp on the desktop PC to plan my (GPX) routes, and Cruiser loads+obeys them just fine.  So if you need to ask about workflow or equivalent tasks between Apps, start a topic and tag me.

**Reply by Maverikk65** [reply]
OMG --- I see my error. Many tnx for pointing me there... On the smartphone I only saw the Germany folder and downloaded the regions. On the tablet, with the much larger screen, I only saw the map (and overlooked the folder) and thought I have to download the 1.8 GB map as a whole... 
Tnx! Danke, VG, NG.

**Reply by Maverikk65** [reply]
> Hello @Maverikk65 , former CoPilot user here - You certainly won't miss having to convert routes and copying files into strange folders if you want to import routes or favourites, Cruiser does it much easily. I also liked CoPilot because of its ability to handle stop/start routes with a photo/rest stop on the countdown, and it was _reasonable_ at making a route on the fly, but again - Cruiser does it much more elegantly.
>>> Hello @MotoUKRider many thanks. Well, I used to plan my routes on the Kurviger website ever since 2016 and finally using the Google drive it was not too complicated, as my file manager kept the destination folder in which the file had to be moved. The most annoying thing for me is/was that two navigations are each trying to calculate the route according to their algorithm 
and this gave unwanted routing... Ending up with 70 shaping points for a 3 hour route is just not desireable... 
 
> There are some things you might miss, but Cruiser isn't a _"Get me there as quickly as possible by telling me about traffic"_ kinda App, there's plenty like that - But it does handle "fun" routes very well, and complex multi-stop tours, different routing profiles (ie: fast or curvy) _within the same_ route are just a few features.
Yes, I know, and I don't need this. I have and use different apps for my driving (actually not a motobike but a Nissan 370Z (without navigation) or a Nissan QQ (with built-in navigation)). Either I just want from A to B on the autobahn as quickly as I can go --- for this I took only Copilot with the traffic infos, which were quite good... If I encounter a road block or serious jam I can just leave the autobahn, make a quick stop, plan 3-5 waypoints and go on... This was quite good on Copilot! Now I have to take another app for this... Maybe TomTom which I tried but did not like, although I believe I have TomTom built-in in the QQ... So in future for those unplanned routes I would take Kurviger app (K3), plan and drive with Cruiser, most likely... 
On the other side I did very nice tours of 2-3 weeks to Italy, France, Spain, Hungary, Poland... Here I don't drive autobahns at all and pre-plan my routes on the pc using the Kurviger website and drive with --- Cruiser, most likely...? Not sure yet. 
The problem is, that on 95% of my pre-planned routes something gets blocked or a change of plans and I believe that the Kurviger app for driving is really very bad at this. Stop the navigation, go into planning mode, and then start navigation again --- it is way too complicated and unflexible. So often I was standing at the road block and quickly deleted 3 waypoints in Copliot and added 4 more and off we go... THAT was a big pro on Copilot, loved it for that. Hate it for breaking and no community not even listen to its users. So bad...

> 
> I use Garmin BaseCamp on the desktop PC to plan my (GPX) routes, and Cruiser loads+obeys them just fine. So if you need to ask about workflow or equivalent tasks between Apps, start a topic and tag me.
>>> Okay, I did have a quick look into Basecamp some years ago, didn't like it instantly and found Kurviger. I normally plan on the pc with two monitors, having Kurviger 2x open on different browsers, on with a big view to overview the whold route and one with detailed planning... ;-)  I liked the export/import of .kurviger files from the Kurviger to the Cruiser for now, using the Google drive... 
 
Danke, VG, NG.

**Reply by devemux86** [reply]
@Maverikk65 

- You can **tap** the direction icon at top left for the **navigation actions** like "Avoid road block" (image 2).

- You can also **long press** the direction icon for the **navigation options** (image 3).

Remember that everything in Cruiser works **offline**: route calculations, avoid road blocks, etc.

<img src="https://github.com/user-attachments/assets/fd19edb6-bb25-4723-908e-8b1715c7b2b9" width="33%"/><img src="https://github.com/user-attachments/assets/d834d989-7139-4a34-b151-2509437fb2fb" width="33%"/><img src="https://github.com/user-attachments/assets/236c7522-c762-4283-994d-4f8f6d3d8711" width="33%"/>
